pulse_actions/handlers/talos.py

```diff
@@ -0,0 +1,70 @@
+"""
+This module is for the following use case:
+
+ - Talos jobs:
+
+    * Trigger talos jobs twice if they are from PGO build.
+"""
+import logging
+
+from mozci import query_jobs
+from pulse_actions.utils.misc import filter_invalid_builders
+from mozci.errors import PushlogError
+from mozci.mozci import trigger_talos_jobs_for_build
+from mozci.platforms import get_buildername_metadata
+from mozci.sources import buildjson
+
+LOG = logging.getLogger()
```

Comment:
I would like to try something new in here to make the logs of pulse_actions more clear.

Could you please use this inside of getLogger()?
```python
LOG = logging.getLogger(__name__.split('.')[-1])
```